cfme/tests/cloud/test_cloud_timelines.py

```diff
@@ -13,9 +15,18 @@
 
 
 pytestmark = [pytest.mark.tier(2),
-              pytest.mark.uncollectif(lambda: version.current_version() < '5.7'),
+              pytest.mark.uncollectif(lambda provider: provider.one_of(EC2Provider) and
```

Comment:
I thought of such a mark but it is neccessary only for one provider. Two other providers aren't long running.